docs/Training-PPO.md

```diff
@@ -14,7 +14,19 @@ For information about imitation learning, which uses a different training algori
 
 Successfully training a Reinforcement Learning model often involves tuning the training hyperparameters. This guide contains some best practices for tuning the training process when the default parameters don't seem to be giving the level of performance you would like.
 
-### Hyperparameters
+## Hyperparameters
+
+#### Gamma
+
+`gamma` corresponds to the discount factor for future rewards. This can be thought of as how far into the future the agent should care about possible rewards. In situations when the agent should be acting in the present in order to prepare for rewards in the distant future, this value should be large. In cases when rewards are more immediate, it can be smaller.
+
+Typical Range: `0.8` - `0.995`
+
+#### Lambda
```

Comment:
Lambd - since presumably the headers should match the variable (?)

Reply:
In the same way the headers have spaces instead of underscores, I think the header should reflect the actual name of the hyperparameter rather than the name of the variable. This also allows people to build intuition about the concepts around RL, rather than just the specifics of how we happened to implement PPO, or name our variables.

Reply:
The leaders do not match necessarily I think.